src/components/app-scope.js

```diff
@@ -57,8 +248,23 @@ class AppScope extends React.Component {
 
 function mapStateToProps (state) {
   return {
-    activeApp: state.activeApp
+    activeApp: state.activeApp,
+    optionsList: state.optionsList
+  }
+}
+
+const mapDispatchToProps = dispatch => {
+  return {
+    SetOptionsList: () => {
+      dispatch(setOptionsList());
+    },
+    ChangeActiveApp: (id, field, data, successCallback, errCallback) => {
+      dispatch(changeActiveApp(id, field, data, successCallback, errCallback))
+    }
   }
 }
 
-export default connect(mapStateToProps)(AppScope)
+export default connect(
+  mapStateToProps, 
+  mapDispatchToProps
+)(AppScope)
```

Comment:
Add a newline

Reply:
All files have similar structure. No newline is present in other files